src/ActionExecutor/UserConfirmation.php

```diff
@@ -238,7 +238,7 @@ protected function jsGetExecute($obj, $id): array
             $this->hide(),
             $this->ok->js(true)->off(),
             $this->cancel->js(true)->off(),
-            $this->hook('afterExecute', [$obj, $id]) ?: $success ?: new jsToast('Success' . (is_string($obj) ? (': ' . $obj) : '')),
+            $this->hook(Basic::HOOK_AFTER_EXECUTE, [$obj, $id]) ?: $success ?: new jsToast('Success' . (is_string($obj) ? (': ' . $obj) : '')),
```

Comment:
Can't we use something like `self::HOOK_AFTER_EXECUTE` or `static::HOOK_AFTER_EXECUTE` here?